spark/dl/src/main/scala/com/intel/analytics/bigdl/parameters/AllReduceParameter.scala

```diff
@@ -207,7 +207,8 @@ class AllReduceParameter[T: ClassTag](
               val blockId = getWeightBlockId(pid)
               val localBuffer = BlockManagerWrapper.getLocalOrRemoteBytes(blockId).getOrElse {
                 throw new RuntimeException(s"Didn't find weight block $blockId in the block " +
-                  s"manager. Did you initialize this AllReduceParameter on every executor?")
+                  s"manager. This is usually caused by executor crashed. Please check your" +
+                  s"executors' log to see the error(usually a OutOfMemory error)")
```

Comment:
see the error (usually a OutOfMemory error)

Reply:
a -> an